src/Propel/Runtime/ActiveQuery/BaseModelCriteria.php

```diff
@@ -28,6 +28,11 @@ class BaseModelCriteria extends Criteria implements IteratorAggregate
      */
     protected $modelTableMapName;
 
+    /**
+     * @var bool
+     */
+    protected $useAliasInSQL = false;
```

Comment:
We try to use same style for acronyms as for the rest of naming: `...InSql`
Is that possible here too?

Reply:
Oh, I see you moved this from somewhere else. So we better keep the same style then for now.

Reply:
Yes, I remember, that was a property defined in child class BaseModelCriteria, but then accessed in the parent class through `static`, and then stan complained about it (rightly so). And even worse, my editor couldn't find it, so I cleaned it up